custom/icds_reports/static/icds_reports/js/spec/adhaar-beneficiary.directive.spec.js

```diff
@@ -212,14 +212,14 @@ describe('Adhaar Beneficiary Directive', function () {
         controller.userLocationId = 'test_id4';
         controller.location = {name: 'name4', location_id: 'test_id4'};
         controller.selectedLocations.push(
-            {name: 'name1', location_id: 'test_id1'},
-            {name: 'name2', location_id: 'test_id2'},
-            {name: 'name3', location_id: 'test_id3'},
-            {name: 'name4', location_id: 'test_id4'},
-            {name: 'name5', location_id: 'test_id5'},
-            {name: 'name6', location_id: 'test_id6'}
+            {name: 'name1', location_id: 'test_id1', user_have_access: 0},
+            {name: 'name2', location_id: 'test_id2', user_have_access: 0},
+            {name: 'name3', location_id: 'test_id3', user_have_access: 0},
+            {name: 'name4', location_id: 'test_id4', user_have_access: 1},
+            {name: 'All', location_id: 'all', user_have_access: 0},
+            null
```

Comment:
Why is this `null` being added?

Reply:
@emord I correct how ```selectedLocation``` variable looks in the test because when we clicking in dashboard we always have ```null``` after the last ```All``` option.